### PR TITLE
New PRS in EMML1832 (no.4)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created new PRS from EMML1832. The last four of them have been discussed in https://github.com/BetaMasaheft/Documentation/issues/2458.

### CODE REVIEW

File: new/PRS14259BahaylaKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <revisionDesc>
+            <change who="CH" when="2024-01-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Baḫāyla Krǝstos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (thea-m) [inline new/PRS14259BahaylaKrestos.xml:49]: fidal lacking, fidal and transcription to be swapped

File: new/PRS14259BahaylaKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    <faith type=""/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14259BahaylaKrestos" passive=""/>
```
Comment (thea-m) [inline new/PRS14259BahaylaKrestos.xml:56]: ms ID

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! As I wrote in the issue, I am not convinced that the last four records here are persons. I think they might refer to places or other entities (e.g. māḫbar).

**Comment by CarstenHoffmannMarburg** [comment]
> Thank you! As I wrote in the issue, I am not convinced that the last four records here are persons. I think they might refer to places or other entities (e.g. māḫbar).

Yes, indeed, this name also sounds strange to me. As far as I have come across, there are a number of personal names that include the name Māḫbar as part of their name:
Tasfa Kokab Maḫbar (PRS9276TasfaKo)
Fǝre Māḫbar (PRS11144FereM, PRS4067FereMah, PRS12413FereMahbar)
Yamāna ʾAb Lāʾka Māḫbar (PRS13175YamanaA)
Takla Māḫbar (PRS11758TaklaMahbar)

I can imagine that the name Māḫbar in these names could be an allusion to the House of Christ = Church or the Temple in Jerusalem. It could also be an abbreviation of a combined name like one of the above.

**Comment by CarstenHoffmannMarburg** [comment]
In this pull request, 6 out of 10 records are OK. But the last four records need to be deleted, because they are assumed to be placenames, that we decided not to mark up in the record. For this reason, this branch needs to be approved and merged and the four problematic records need to be deleted in a new pull request afterwards.

**Comment by eu-genia** [comment]
I deleted
PRS14261Mahbar
PRS14262Zengita
PRS14263Wambaya
PRS14264Yalaela
on the backend so they can already be deleted in this branch

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**